discovery-provider/src/challenges/challenges.json

```diff
@@ -5,5 +5,13 @@
 		"amount": "5",
 		"active": true,
 		"step_count": 7
+	},
+	{
+		"id": "track-upload",
+		"type": "numeric",
+		"amount": "5",
+		"active": true,
+		"step_count": 3,
+		"starting_block": 20000000
```

Comment:
FYI etherscan has a nice estimator https://etherscan.io/block/countdown/20000000

Reply:
this is fire